### PR TITLE
[PBA-6208] Added presented viewcontrollers support

### PR DESCRIPTION
**Pull request by IvanSakharovskii**

### CODE REVIEW

File: sdk/iOS/OoyalaSkinSDK/ViewControllers/PresentedViewControllerSupport/PresentedViewControllerHelper.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//
+// Created by Ivan Sakharovskii on 6/6/18.
+// Copyright (c) 2018 ooyala. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
```
Comment (MarkEPAM) [inline sdk/iOS/OoyalaSkinSDK/ViewControllers/PresentedViewControllerSupport/PresentedViewControllerHelper.h:7]: add 1 space

File: sdk/iOS/OoyalaSkinSDK/ViewControllers/PresentedViewControllerSupport/PresentedViewControllerHelper.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+//
+// Created by Ivan Sakharovskii on 6/6/18.
```
Comment (MarkEPAM) [inline sdk/iOS/OoyalaSkinSDK/ViewControllers/PresentedViewControllerSupport/PresentedViewControllerHelper.h]: remove name

**Comment by akshaydch11** [comment]
Holding this PR for merge until 4.36 goes out. We will merge it for next release.

**Review by brol1dev — APPROVED** [review]
Tested it on my side, works fine 👍

**Comment by akshaydch11** [comment]
LGTM

**Pull request merged**